modAionImpl/src/org/aion/zero/impl/sync/TaskImportBlocks.java

```diff
@@ -292,9 +299,11 @@ static boolean isAlreadyStored(AionBlockStore store, Block block) {
 
     private ImportResult importBlock(Block b, String displayId, SyncMode mode) {
         ImportResult importResult;
-        long t1 = System.currentTimeMillis();
+        long t1 = System.nanoTime();
         importResult = this.chain.tryToConnect(b);
-        long t2 = System.currentTimeMillis();
+        long t2 = System.nanoTime();
+        long DIVISOR_MS = 1000000L;
```

Comment:
minor:  `1000000L` -> `1_000_000L` (makes the numbers easier to read); same for the `1000L` below